buildhat/serinterface.py

```diff
@@ -138,6 +140,7 @@ def __init__(self, firmware, signature, version, device="/dev/serial0", debug=Fa
                 else:
                     self.write(b"version\r")
 
+        self.state = HatState.FIRMWARE
```

Comment:
From the commit message, does this need to be removed?

Reply:
Good point, just dropped